src/lib/styles/themes/dark.scss

```diff
@@ -10,6 +10,10 @@
   --text-text: var(--purple-900);
   --input-focus-background: var(--purple-dark-450);
   --button-primary: var(--purple-dark-accent);
+  --table-background: var(--purple-dark-550);
+  --table-header-background: var(--purple-dark-450);
+  --table-row-background: var(--purple-dark-550);
+  --table-row-background-hover: var(--purple-dark-500);
```

Comment:
I think for background colors, `background` should be the suffix, so `hover` should come before it.
For example, there is also `--button-card-focus-background` where `focus` comes before `background`.

Reply:
I don't have a strong opinion, but I agree that we should have consistent naming conventions.